### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by iulianoroberto** [bitcraze/discussions#756] [Q&A]

**Comment by tobbeanton** [comment]
It is most likely the soldering of the sensor board that is broken. Perhaps after a crash or damaged during shipping. This is usually easily fixable with a soldering iron by just melting the solder on each pad that is soldering the sensor board to the main board.

**Comment by krichardsson** [comment]
@robertoiuliano98 Would it be possible for you to restore the question? The discussion thread does not make a lot of sense to other people if the question is removed. Thanks!
